src/platform/android/KeyValueStoreManagerImpl.h

```diff
@@ -61,7 +58,7 @@ class KeyValueStoreManagerImpl : public KeyValueStoreManager
  * Chip applications should use this to access features of the KeyValueStoreManager object
  * that are common to all platforms.
  */
-inline KeyValueStoreManager & KeyValueStoreMgr()
+inline KeyValueStoreManager & KeyValueStoreMgr(void)
```

Comment:
I think we prefer to not specify void here. Same below

Reply:
fixed at 21c564d